core/contracts/financial-templates/implementation/ExpiringMultiPartyCreator.sol

```diff
@@ -60,19 +82,43 @@ contract ExpiringMultiPartyCreator is ContractCreator, Testable {
         constructorParams.isTest = isTest;
         constructorParams.finderAddress = finderAddress;
 
+        // Enforce configuration constrainments
+        require(params.expirationTimestamp <= LATEST_EXPIRATION_TIMESTAMP);
+        require(params.disputeBondPct.isGreaterThan(MIN_DISPUTE_BOND_PCT));
+        require(!_stringsEqual(params.syntheticName, STRING_NULL));
```

Comment:
Can you check the length of the string is nonzero instead?

Reply:
Yeah that's prob cleaner. I was thinking _stringsEqual would be useful in future but that's outside this PR scope

Reply:
done